scripts/build/build/targets.py

```diff
@@ -362,6 +363,18 @@ def K32WTargets():
     yield target.Extend('lock-low-power-release', app=K32WApp.LOCK, low_power=True, release=True).GlobBlacklist("Only on demand build")
 
 
+def Cyw30739Targets():
+    target = Target('cyw30739', Cyw30739Builder)
+
+    board_targets = [
+        target.Extend('cyw930739m2evb_01',
+                      board=Cyw30739Board.CYW930739M2EVB_01)
+    ]
+
+    for board_target in board_targets:
+        yield board_target.Extend('light', app=Cyw30739App.LIGHT)
```

Comment:
nit: I think you can do in one or not  have it a generator. Like:

```python
yield Target('cyw30739-cyw930739m2evb_01-light', Cyw30739Builder, board=Cyw30739Board.CYW930739M2EVB_01, app=Cyw30739App.LIGHT)
```

also your app and boards are the default so you can do:

```python
yield Target('cyw30739-cyw930739m2evb_01-light', Cyw30739Builder)
```